clamav.py

```diff
@@ -69,13 +69,21 @@ def update_defs_from_s3(s3_client, bucket, prefix):
                 print("Not downloading %s because local md5 matches s3." % filename)
                 continue
             if s3_md5:
-                to_download[file_prefix] = {
```

Comment:
This is a nasty bug.  By using the file_prefix as the key you could not download daily.cld and daily.cvd in the same run.  The first run in a lambda instance would be missing daily.cld, then the second run would get it since daily.cvd was already present.